### ISSUE TITLE
[Accessibility]: Missing aria-label for 2 svg elements in LineChart component

### ISSUE THREAD

**Issue by David-Yuen** [carbon-design-system/carbon-charts#1949]
### Application/Team

Data Replication

### Environment

Browser

### What happened? What did you expect to happen instead?

When rendering the `LineChart` component, there are 2 `svg` elements that are missing `aria-label`.

```
<svg
    height="130"
    width="282.21875"
    y="0"
    x="0"
    role="presentation"
    class="chart-grid-backdrop">
```

```
<svg
    height="100%"
    width="100%"
    role="presentation"
    class="cds--cc--zero-line">
```

I'm seeing the following accessibility scan errors (same for both):
```
Rule ID: svg_graphics_labelled
Reason ID: fail_acc_name
```

![Image](https://github.com/user-attachments/assets/90a76512-7197-43a0-a1f6-078fc78ec3dc)

![Image](https://github.com/user-attachments/assets/6a2675e9-fe89-4a9c-ab16-4489bd3eb48f)

### What WCAG 2.1 checkpoint does the issue violate?

svg_graphics_labelled

### Version

"@carbon/charts-react": "1.22.7"

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P3
